lib/src/Base/Graph/Contour.cxx

```diff
@@ -226,6 +437,50 @@ void Contour::load(Advocate & adv)
   adv.loadAttribute( "levels_", levels_ );
   adv.loadAttribute( "labels_", labels_ );
   adv.loadAttribute( "drawLabels_", drawLabels_ );
+  if (adv.hasAttribute("isFilled_"))
+    adv.loadAttribute("isFilled_", isFilled_);
+  else
+    isFilled_ = ResourceMap::GetAsBool("Contour-IsFilled");
```

Comment:
no need to set the default value here: it is set already by the default ctor

Reply:
OK, done.